File: frontend/src/types/navigation.ts
export type MenuItem = {
    key: string;
    title: string;
    icon: string;
    url_name: string;
    perms: string[];
    active_patterns: string;
    url: string;
    active: boolean;
}

export interface DockItem {
    nombre: string;
    icon: string;
    url_name: string;
    args?: any[];
    active_patterns?: string[];
    active: boolean;
    exact?: boolean;
    perms?: string[];
}

export type BreadcrumbItem = {
    label: string;
    url?: string;
    icon?: string;
};

File: apps/evidencias_moldes/navigation.py
def build_evidencias_moldes_dock():
    return [
        {
            "nombre": "Inicio",
            "icon": "icon-[heroicons--home]",
            "url_name": "home",
            "active_patterns": ["home"],
            "exact": True,
        },
        {
            "nombre": "Evidencias",
            "icon": "icon-[tabler--address-book]",
            "url_name": "evidencias_moldes:root",
            "active_patterns": ["evidencias-moldes:"],
            "perms": ["evidencias_moldes.acceder_explorador_direccion_obras"],
        },
    ]